src/modules/Plans/PlanRow.tsx

```diff
@@ -69,6 +71,7 @@ const StatusCell = ({ entity: { status, type, vmCount, vmDone, name, object } }:
     return (
       <Link name={name} isInline={false}>
         <Progress
+          id={`progress_for_${name}_in_${namespace}`}
```

Comment:
progerss is missing the ouiaid :-(

Reply:
`Progress` component seems to require the `id` ...if you want to use the `title` :) 
The formula is the following:
```
id = this.props.id || getUniqueId();
// ...
 if (title || ariaLabelledBy) {
      progressBarAriaProps['aria-labelledby'] = title ? `${this.id}-description` : ariaLabelledBy;
    }
```